cs_insights_prediction_endpoint/models/generic_model.py

```diff
@@ -1,99 +1,99 @@
-"""This modulew implements the generic-model"""
+"""This module implements the generic-model"""
 from datetime import datetime
 from pathlib import Path
 from typing import Any, Dict, Optional, TypeVar
 
 from pydantic import BaseModel, Field
 
-T = TypeVar("T", bound="GenericModel")
+T = TypeVar("T", bound="generic_model")
 
 
-class GenericModel(BaseModel):
+class generic_model(BaseModel):
```

Comment:
This is not how you reference classes in pep8/black. Please refer to the documentation...

https://peps.python.org/pep-0008/
https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html